README.md

```diff
@@ -2,24 +2,20 @@
 [![Build
 Status](https://travis-ci.org/helm/monocular.svg?branch=master)](https://travis-ci.org/helm/monocular)
 
-Monocular is web-based UI for managing Kubernetes applications packaged as Helm
-Charts. It allows you to search and discover available charts from multiple
-repositories, and install them in your cluster with one click.
+Monocular is a web-based UI and API that enables the search and discovery of
+charts from multiple Helm Chart repositories. It is the codebase that powers the
```

Comment:
powers? will power?

Reply:
If we say "will power" we'll need to go and update this later, I think it's okay to just say powers here.